### PR TITLE
Update th.m3u (Some channels)

### PR DESCRIPTION
**Pull request by winnie3101**

### CODE REVIEW

File: streams/th.m3u [hunk 1]
```diff
@@ -9,10 +9,10 @@ https://thaipbs-ujxrch.cdn.byteark.com/live/playlist.m3u8
 https://thaipbs-ujxrch.cdn.byteark.com/live/playlist_1080p/index.m3u8
 #EXTINF:-1 tvg-id="Channel5.th" status="online",Channel 5 (480p)
 https://tc-live1.sanook.com/live/22302_ch5.m3u8
-#EXTINF:-1 tvg-id="Channel5.th" status="timeout",Channel 5 (1080p)
-http://110.170.117.27:1935/apptv5hd1live/vdo-tv5hd1/playlist.m3u8
-#EXTINF:-1 tvg-id="Channel7.th" status="blocked",Channel 7 (240p)
-https://bcovlive-a.akamaihd.net/2d37038b355f4ea6a6b0d46993dc285c/ap-southeast-1/5282994675001/profile_0/chunklist.m3u8
+#EXTINF:-1 tvg-id="Channel5.th" status="online",Channel 5 (1080p)
+https://639bc5877c5fe.streamlock.net/tv5hdlive/tv5hdlive/chunklist_w555179815.m3u8
```
Comment (Carlinhos027) [inline streams/th.m3u:13]: Change the chunklist part and write playlist instead

**Review by Carlinhos027 — CHANGES_REQUESTED** [review]

File: streams/th.m3u (hunk 1 above)
Comment (winnie3101) [reply streams/th.m3u:13]: Thank you ^^

Comment (BellezaEmporium) [inline streams/th.m3u:13]:
```suggestion
https://639bc5877c5fe.streamlock.net/tv5hdlive/tv5hdlive/playlist.m3u8
```

**Review by BellezaEmporium — APPROVED** [review]
Good to go.

**Review by Carlinhos027 — APPROVED** [review]

**Pull request merged**